src/js/tabs/blackbox.js

```diff
@@ -297,6 +308,8 @@ tab.initialize = function (callback) {
             FC.BLACKBOX.blackboxDenom = parseInt(RatesSelect.val(), 10);
             FC.DEBUG_CONFIG.debugMode = parseInt(debugModeSelect.val(), 10);
             FC.DEBUG_CONFIG.debugAxis = parseInt(debugAxisSelect.val(), 10);
+            FC.BLACKBOX.blackboxInitialEraseKiB = parseInt(initialEraseInput.val(), 10) * 1024;
+            FC.BLACKBOX.blackboxRollingErase = rollingEraseCheckbox.prop('checked')? 1 : 0;
```

Comment:
`? 1 : 0` is not necessary. `true` and `false` coerce to and from `1` and `0`.

Reply:
Changed to Number() to keep it as a number.